analyses/0007-porter.md

```diff
@@ -1,19 +1,10 @@
 ---
-title: Porter tech docs assessment draft
-tags: porter
+cSpell:ignore: Uchechukwu Obasi
 ---
 
-# Notes
+<!-- markdownlint-disable no-duplicate-heading -->
 
-meeting notes:
-https://docs.google.com/document/d/12OGtSaUtlc7OA_iPnUvmVaiKg8yM7_QBzFnMgoHvnLw/edit?usp=sharing
-website: https://getporter.org/ repos:
-
-- https://github.com/getporter/porter (main site), site content is in docs/
-  folder
-- https://github.com/getporter/operator (https://getporter.org/operator site)
-
-# Porter Docs Assessment
+# Porter Docs Analysis
```

Comment:
@nate-double-u : I actually dropped the front-matter title given that this in-page title takes precidence.